aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/s3/S3ExpressDecorator.kt

```diff
@@ -201,3 +197,25 @@ class S3ExpressIdentityProviderConfig(codegenContext: ClientCodegenContext) : Co
             }
         }
 }
+
+private fun inlineModule(runtimeConfig: RuntimeConfig) =
+    RuntimeType.forInlineDependency(
+        InlineAwsDependency.forRustFile(
+            "s3_express",
+            additionalDependency = s3ExpressDependencies(runtimeConfig).toTypedArray(),
+        ),
+    )
+
+private fun s3ExpressDependencies(runtimeConfig: RuntimeConfig) =
+    listOf(
```

Comment:
you can change this to `arrayOf` and skip the `toTypedArray` above

Reply:
Thanks! Updated in d281d49.